src/ensembl/src/content/app/entity-viewer/components/transcript-filter/EntityViewerTranscriptFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import Checkbox from 'src/shared/components/checkbox/Checkbox';
+
+import styles from './EntityViewerTranscriptFilter.scss';
+
+export type Option = {
+  value: string;
+  label: string;
+};
```

Comment:
as the component is written, the Option type begs to have an `isSelected` field; then the `selectedValues` prop becomes unnecessary.

Reply:
Would that mean we will have to pass the entire set of options back to the parent component onChange and let it filter for the selected values?